_backend/bootstrap.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * _backend/bootstrap.php
+ * Entry file for the whole website. Handles configuration loading.
+ */
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+// TODO: why are these not in composer?
+require_once __DIR__ . '/../docs/_mdr/Parsedown.php';
+require_once __DIR__ . '/../docs/_mdr/ParsedownExtra.php';
```

Comment:
Moved to composer, so long as we're sure they'll still load everywhere where docs need them.